src/config.rs

```diff
@@ -2162,6 +2190,11 @@ impl TiKvConfig {
             self.raft_store.raftdb_path = config::canonicalize_path(&self.raft_store.raftdb_path)?;
         }
 
+        let default_er_path = config::canonicalize_sub_path(&self.storage.data_dir, "raft-engine")?;
+        if self.raft_engine.config.dir.is_empty() {
+            self.raft_engine.config.dir = default_er_path;
+        }
+
```

Comment:
Should we also do some checks like https://github.com/tikv/tikv/blob/a562e0de60aa6e73b1796ad8d9a7b45287401044/src/config.rs#L2165-L2176 and https://github.com/tikv/tikv/blob/a562e0de60aa6e73b1796ad8d9a7b45287401044/src/config.rs#L2315